services/frontend/domain/usecases/process_manifest_files.go

```diff
@@ -71,39 +69,38 @@ func (u *Usecases) ProcessManifestFiles(manifestDir string) error {
 			if rawManifestData, processingError = os.ReadFile(manifestFilepath); processingError != nil {
 				return
 			}
-
 			if processingError = yaml.Unmarshal(rawManifestData, &unmarshalledManifest); processingError != nil {
 				return
 			}
```

Comment:
I've noticed that `yaml.Unmarshal(rawManifestData, &unmarshalledManifest)` doesn't quite make sense. Since `unmarshalledManifest` is already a pointer `*manifest.Manifest`, can you change it to just `yaml.Unmarshal(rawManifestData, unmarshalledManifest)`